File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--protect-parts-of-documents-by-using-content-controls_4.vb
    Dim groupControl1 As Microsoft.Office.Tools.Word.GroupContentControl

    Private Sub ProtectFirstParagraph()

        Dim VstoDocument As Microsoft.Office.Tools.Word.Document = _
            Globals.Factory.GetVstoObject(Me.Application.ActiveDocument)

        VstoDocument.Paragraphs(1).Range.InsertParagraphBefore()

        Dim range1 As Word.Range = VstoDocument.Paragraphs(1).Range
        range1.Text = "You cannot edit or change the formatting of text " & _
                "in this paragraph, because this paragraph is in a GroupContentControl."
        range1.Select()

        groupControl1 = VstoDocument.Controls.AddGroupContentControl("groupControl1")
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--protect-parts-of-documents-by-using-content-controls_4.cs
        private Microsoft.Office.Tools.Word.GroupContentControl groupControl1;

        private void ProtectFirstParagraph()
        {
            Microsoft.Office.Tools.Word.Document vstoDocument =
                Globals.Factory.GetVstoObject(this.Application.ActiveDocument);

 
            vstoDocument.Paragraphs[1].Range.InsertParagraphBefore();

            Word.Range range1 = vstoDocument.Paragraphs[1].Range;
            range1.Text = "You cannot edit or change the formatting of text " +
                "in this sentence, because this sentence is in a GroupContentControl.";
            range1.Select();

            groupControl1 = vstoDocument.Controls.AddGroupContentControl("groupControl1");
        }